typeDefinition/PositionChain.d.ts

```diff
@@ -46,4 +46,4 @@ interface PositionEmptyChain extends PositionBaseEvalChain<PositionEmptyChain> {
 interface PositionAbandonedChain extends AbstractChain {}
 
 type PositionQueryResult = string;
```

Comment:
is it correct? for me 
const el = await suitest.position(10, 10);
		console.log(typeof el); // prints `boolean`

Reply:
I think, there is no query for position, only cookie, element, location and jsExpression.